modules/index.js

```diff
@@ -0,0 +1,109 @@
+import {
+  lcontact, contact21, section31, list11,
+} from './contact.js';
+import {
+  add, contact2, section21, list1,
+} from './add.js';
+import {
+  list, lList, contact, section2,
+} from './list.js';
+
+class Awesomebooks {
+  constructor() {
+    this.data = JSON.parse(localStorage.getItem('data')) || [];
+  }
+
+  add() {
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly your navigation items are not working at the movment and can not add `Book` items(`add` and `remove` functions), please fix them 🙏🏼 

![mobile](https://github.com/enocol/AwesomeBooks/assets/63915024/519c3ead-f20e-439d-a383-8df03755a76d)

Reply:
@mukhammadrizooff ohh Thats strange. Let me recheck. Everything was working fine when I submit for a review.